inc/editor/patterns/map-info.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Block pattern for a map-info group.
+ */
+
+namespace WMF\Reports\Editor\Patterns\MapInfo;
+
+const NAME = 'wmf-reports/map-info';
+
+const PATTERN = <<<CONTENT
+<!-- wp:group {"align":"full","backgroundColor":"wmf-report-bright-yellow","className":"map-info","layout":{"type":"constrained"}} -->
```

Comment:
I think it might be nice to add a "name" to each of the patterns we create here, to make it easier to navigate around the editor. No need to hold up this PR, we can come back to that later.

![image](https://github.com/wikimedia/wikimedia-wordpress-annual-report-plugin/assets/665992/4428d491-4028-4dfc-b439-3d052ddaeeed)

Reply:
Great idea! I'll fix this :)